spec/classes/pulp_admin_spec.rb

```diff
@@ -4,7 +4,7 @@
   context 'on RedHat' do
     context 'install class without parameters' do
       let :facts do
-        on_supported_os['redhat-7-x86_64'].merge(:concat_basedir => '/tmp', :mongodb_version => '2.4.14', :root_home => '/root')
+        on_supported_os['redhat-7-x86_64'].merge(:concat_basedir => '/tmp', :root_home => '/root')
```

Comment:
We can drop the whole merge since we set these facts in ```spec_helper.rb```. Doesn't have to be part of this PR. Just an observation.